org.mozilla.Firefox.json

```diff
@@ -15,6 +15,8 @@
         "--socket=wayland",
         "--socket=x11",
         "--filesystem=home",
+        "--filesystem=/var/lib/codecs:ro",
```

Comment:
I highly doubt this will do anything – AFAIK this directory is Endless-specific, so this pristine Mozilla-supplied binary won't do anything with it.

Reply:
Mozilla dlopens libavcodec, so placing it on the library path might actually work provided our libcs are sufficiently compatible - however I'm not sure what happens if the runtime-provided libavcodec has a different, higher soname than the one provided by the OS. Presumably AAC and H.264 disappear in this case. Considering we're trying to move to free implementations etc - it doesn't seem that worthwhile.